docs/static/breaking-changes.asciidoc

```diff
@@ -1,93 +1,256 @@
 [[breaking-changes]]
 == Breaking changes
 
-This section discusses the changes that you need to be aware of when migrating your application to Logstash {version}.
+This section discusses the changes that you need to be aware of when migrating your application to Logstash {version} from the previous major release of Logstash (2.x).
+
+[float]
+=== When to Upgrade
+
+Fresh installations can and should start with the same version across the Elastic Stack.
+
+Elasticsearch 5.0 does not require Logstash {version}. An Elasticsearch 5.0 cluster will happily receive data from a
```

Comment:
Yeah, I kind of wonder if I should replace `{version}` with literally `5.0` (or even more generally `5.x`?) in the breaking changes doc, since that is really a constant.